pkgs/tools/filesystems/dwarfs/version_info.patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/include/dwarfs/version.h b/include/dwarfs/version.h
+new file mode 100755
+index 0000000..9b12c59
+--- /dev/null
++++ b/include/dwarfs/version.h
+@@ -0,0 +1,16 @@
++// autogenerated code, do not modify
```

Comment:
@keksbg @tpwrules How does one generate it, and when should one?

The `default.nix` would really benefit from some comments on what is going on.

Trying to upgrade to a newer version, the build fails with `'PRJ_GIT_DATE' was not declared in this scope` and there isn't a good explanation of what needs to be done about it.

@tpwrules What about the special flags upstream made for us in https://github.com/mhx/dwarfs/issues/155 ?

Should those be used?

Reply:
I did not think it would need to be regenerated, it's autogenerated by the build system and we fill it in separately. But I guess its structure did change. We should migrate to the special flag then.

I can look at it myself in a day or two and ask you to review, or you can start a draft PR and I can weigh in.

Reply:
Doing it in PR #321996